docs/release.md

```diff
@@ -7,15 +7,18 @@
 
 # Release Notes
 
-Type | desc. | version
+Tyep | desc. | version
```

Comment:
typo

Reply:
fixed in fc02e086585bc7cad79071a619769d5940dcff24 .